src/ol/format/gml/gml3format.js

```diff
@@ -1332,27 +1332,3 @@ ol.format.GML3.prototype.writeFeaturesNode = function(features, opt_options) {
  * @api stable
  */
 ol.format.GML = ol.format.GML3;
-
-
-/**
- * Encode an array of features in GML 3.1.1 Simple Features.
- *
- * @function
- * @param {Array.<ol.Feature>} features Features.
- * @param {olx.format.WriteOptions=} opt_options Options.
- * @return {string} Result.
- * @api stable
- */
-ol.format.GML.prototype.writeFeatures;
```

Comment:
Are we going to need another method of [documenting this method](http://openlayers.org/en/v3.14.1/apidoc/ol.format.GML3.html#writeFeatures)?